testnets/nilliontestnet/assetlist.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "nilliontestnet",
+  "assets": [
+    {
+      "description": "NIL - the native token of Nillion",
+      "denom_units": [
+        {
+          "denom": "unil",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "nil",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "unil",
+      "name": "Nillion",
+      "display": "nil",
+      "symbol": "NIL",
+      "logo_URIs": {
+        "png": ""
+      },
+      "images": [
+        {
+          "png": ""
```

Comment:
Why was the image URI removed?

Reply:
I thought you meant from both files.  I have reinstated.